sympy/utilities/tests/test_wester.py

```diff
@@ -987,7 +987,7 @@ def test_M23():
 def test_M24():
     # TODO: Replace solve with solveset, as of now test fails for solveset
     solution = solve(1 - binomial(m, 2)*2**k, k)
-    answer = log(2/(m*(m - 1)), 2)
+    answer = log(1/binomial(m, 2))/log(2)
```

Comment:
I think you might've accidentally forgot the last commit?

Reply:
hmm...not sure how that happened but yes, it was missed. Thanks.